src/config.ts

```diff
@@ -22,7 +22,8 @@ export default {
     level: 'info'
   },
   general: {
-    resourcePath: 'CDS-Library/CRD-DTR'
+    resourcePath: 'src/cds-library/CRD-DTR',
```

Comment:
Do we want to put the cds-library in the src folder? or at the root of the project? Is it source code? Probably?

Reply:
either way is fine by me